startup/setup_aws.rb

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env ruby
+# Only allow this to run in development
+return unless ENV["EXERCISM_ENV"]=="development"
```

Comment:
I'd like to not refer to the actual ENV vars other than in the specific config place, so pls move this below the `requires` and change to:
```ruby
return unless Exercism.config.environment == :development
```

(I think that's the correct code from memory)

Reply:
Nope.

```
return unless Exercism.environment == :development
```